src/router.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+
+const router = express.Router();
+
+router.get('/', (req, res) => {
+  res.send('hello world');
```

Comment:
This response isn't necessary as the home route is handled in the frontend. You can change this to the * route that sends the react index.html back as @m4v15 mentioned last week. Here's an example: 
```node
app.get('*', (req, res) => {
    res.sendFile(path.join(__dirname, '..', 'client', 'build', 'index.html'));
  })
```